docs/FAQs.md

```diff
@@ -13,6 +13,131 @@ After you set up a repository, you can substitute in any tools you'd like.
 
 If you think the tool would be broadly useful to most consumers of this template, feel free to [file a feature request](https://github.com/JoshuaKGoldberg/create-typescript-app/issues/new?assignees=&labels=type%3A+feature&projects=&template=03-feature.yml&title=%F0%9F%9A%80+Feature%3A+%3Cshort+description+of+the+feature%3E) to add it in.
 
+## Can I create a GitHub action with CTA?
+
+Yes! If you want to read in detail about GitHub Actions [try this](https://docs.github.com/en/actions/creating-actions).
+Here we'll outline the steps required to migrate a CTA app to a GitHub Action:
+
+1. The GitHub Actions does not cater for tabs well, so you will likely want to set your `README.md` to use spaces, not tabs.
+2. GitHub Actions are _not_ like packages published to npm.
+   The output is expected to live in the `dist` folder of a GitHub repo.
+   As a consequence we should:
+
+   - delete `.github/workflows/release.yml` and `.github/workflows/post-release.yml`.
+   - update `.github/workflows/build.yml`
+
+   ```diff
+   -      - run: node ./lib/index.js
+   ```
+
+   - GitHub Actions have a different build mechanism.
+     So you'll be removing `tsup` in favour of [`ncc`](https://github.com/vercel/ncc) which compiles output into a single JS file.
```

Comment:
[Proofreading] _"a different build mechanism"_ is vague. Someone coming in fresh could be confused by that.

I think what this is trying to convey is that GHAs need all built output in one file rather than dependencies. So how about something like this?

```suggestion
   - GitHub Actions run without installing package dependencies.
     Replace `tsup` with [`ncc`](https://github.com/vercel/ncc) to build source files and dependencies into a single JS file.
```